### PR TITLE
Update manageengine_deviceexpert_traversal.rb

### PR DESCRIPTION
**Pull request by jnqpblc**
Changed print_status on line 80 to print_good and added a check for /scheduleresult.de with a return is not a 200 status code.

### CODE REVIEW

File: modules/auxiliary/scanner/http/manageengine_deviceexpert_traversal.rb [hunk 1]
```diff
@@ -45,6 +45,15 @@ def run_host(ip)
     traverse = "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\"
     filename = datastore['FILEPATH']
 
+    res = send_request_raw({
+      'uri' => "/scheduleresult.de",
+      'method' => 'GET'
+    }, 25)
+
+    if res && res.code != 200
+       return
```
Comment (bcoles) [inline modules/auxiliary/scanner/http/manageengine_deviceexpert_traversal.rb]: Some sort of user feedback would be nice, similar to the error checking below.

File: modules/auxiliary/scanner/http/manageengine_deviceexpert_traversal.rb [hunk 2]
```diff
@@ -45,6 +45,18 @@ def run_host(ip)
...
+    else
```
Comment (bcoles) [inline modules/auxiliary/scanner/http/manageengine_deviceexpert_traversal.rb]: I think you meant:

```ruby
    if res && res.code != 200
      print_error("Target is not ManageEngine DeviceExpert")
      return
end
```

?

Comment (jnqpblc) [reply modules/auxiliary/scanner/http/manageengine_deviceexpert_traversal.rb]: Yes, I did.

**Comment by wchen-r7** [comment]
I hope I still have this vuln app. I'll try to verify it.

**Comment by bcoles** [comment]
> 
> 
> I hope I still have this vuln app. I'll try to verify it.

You should be able to retrieve the software from the web archive as per links here:

* https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules/auxiliary/scanner/http/manageengine_deviceexpert_user_creds.md

**Comment by wchen-r7** [comment]
What would I do without @bcoles? Thank you so much!

**Comment by wchen-r7** [comment]
# Release Notes

This adds a check to make sure the web resource exists before performing the directory traversal attack. Also, it will check if the file to loot is found or not.

**Pull request merged**